### PR TITLE
Fixed most pep8 errors

### PR DESCRIPTION
**Pull request by ArseniyAntonov**
Some errors still in code, refactoring may affect functionality

### CODE REVIEW

File: lib/admin_connection.py [hunk 1]
```diff
@@ -50,10 +52,12 @@ def _new_connection(self):
             # tarantool/gh-1163
             # 1. raise only if handshake is not full
             # 2. be silent on crashes or if it's server.stop() operation
-            print 'Handshake error {\n', handshake, '\n}'
+            print
+            'Handshake error {\n', handshake, '\n}'
```
Comment (Totktonada) [inline lib/admin_connection.py]: Don’t get that fix.

File: lib/box_connection.py [hunk 2]
```diff
@@ -77,12 +79,14 @@ def execute_no_reconnect(self, command, silent=True):
         if not command:
             return
         if not silent:
-            print command
+            print
+            command
```
Comment (Totktonada) [inline lib/box_connection.py]: The same question here.

File: lib/box_connection.py [hunk 3]
```diff
@@ -77,12 +79,14 @@ def execute_no_reconnect(self, command, silent=True):
...
         cmd = command.replace(SEPARATOR, ' ') + SEPARATOR
         response = self.py_con.call(cmd)
         result = str(response)
         if not silent:
-            print response
+            print
+            response
```
Comment (Totktonada) [inline lib/box_connection.py]: The same question here.

File: lib/box_connection.py [hunk 4]
```diff
@@ -91,8 +95,10 @@ def execute(self, command, silent=True):
     def call(self, command, *args):
         if not command:
             return
-        print 'call ', command, args
+        print
+        'call ', command, args
```
Comment (Totktonada) [inline lib/box_connection.py]: The same question here.

File: lib/box_connection.py [hunk 5]
```diff
@@ -91,8 +95,10 @@ def execute(self, command, silent=True):
...
         response = self.py_con.call(command, *args)
         result = str(response)
-        print result
+        print
+        result
```
Comment (Totktonada) [inline lib/box_connection.py]: The same question here.

File: lib/pytap13.py [hunk 6]
```diff
@@ -207,7 +209,9 @@ def parse(self, source):
     t.parse(input)
 
     import pprint
+
     for test in t.tests:
-        print test.result, test.id, test.description, "#", test.directive, test.comment
+        print
+        test.result, test.id, test.description, "#", test.directive, test.comment
```
Comment (Totktonada) [inline lib/pytap13.py]: The same question here.

File: lib/tarantool_server.py [hunk 7]
```diff
@@ -784,13 +795,15 @@ def read_pidfile(self):
     def test_option_get(self, option_list_str, silent=False):
         args = [self.binary] + shlex.split(option_list_str)
         if not silent:
-            print " ".join([os.path.basename(self.binary)] + args[1:])
+            print
+            " ".join([os.path.basename(self.binary)] + args[1:])
```
Comment (Totktonada) [inline lib/tarantool_server.py]: The same question here.

File: lib/tarantool_server.py [hunk 8]
```diff
@@ -784,13 +795,15 @@ def read_pidfile(self):
...
         output = subprocess.Popen(args, cwd=self.vardir, stdout=subprocess.PIPE,
                                   stderr=subprocess.STDOUT).stdout.read()
         return output
 
     def test_option(self, option_list_str):
-        print self.test_option_get(option_list_str)
+        print
+        self.test_option_get(option_list_str)
```
Comment (Totktonada) [inline lib/tarantool_server.py]: The same question here.

File: lib/test_suite.py [hunk 9]
```diff
@@ -126,9 +127,10 @@ def gen_server(self):
         try:
             return Server(self.ini, test_suite=self)
         except Exception as e:
-            print e
+            print
+            e
```
Comment (Totktonada) [inline lib/test_suite.py]: The same question here.

File: lib/worker.py [hunk 10]
```diff
@@ -185,7 +190,7 @@ def sigterm_handler(self, signum, frame):
     def __init__(self, suite, _id):
         self.sigterm_received = False
         signal.signal(signal.SIGTERM, lambda x, y, z=self:
-                      z.sigterm_handler(x, y))
+        z.sigterm_handler(x, y))
```
Comment (Totktonada) [inline lib/worker.py]: Such way indent is misleading, IMHO. Is there other way to fix it?

File: lib/admin_connection.py (hunk 1 above)
Comment (ArseniyAntonov) [reply lib/admin_connection.py]: Ugh, sorry, will change it back

**Comment by Totktonada** [comment]
Seems to be obsoleted for now. @sergw iteratively perform code reformattig within feature / bug patch sets.

**Pull request closed without merging**